README.md

```diff
@@ -18,6 +18,7 @@ A directory of ember-codemods
 | [ember-memory-leaks-codemod](https://github.com/ember-codemods/ember-memory-leaks-codemod) | Identifies and fixes common memory leaks in Ember apps | General | |
 | [ember-cli-htmlbars-inline-precompile-codemod](https://github.com/ember-codemods/ember-cli-htmlbars-inline-precompile-codemod) | Transforms imports to avoid using `htmlbars-inline-precompile` | General | |
 | [tagless-ember-components-codemod](https://github.com/ember-codemods/tagless-ember-components-codemod) | Transforms your components to use `tagName: ''` | Components | |
+| [ember-no-get-with-default](https://github.com/steventsao/ember-no-get-with-default-codemod) | Transforms `getWithDefault` with `get` and ternary operator  | General | ember? |
```

Comment:
Would you be interested in moving the codemod into this org? I’m happy to add you as an owner here. 

I’ve personally found it really helpful to have other codemod authors to help triage and whatnot...

Reply:
Yeah, that works for me!

Reply:
Just sent the invite! (Should be able to accept at https://github.com/ember-codemods also)

Reply:
Submitted. Closing this thread =)